src/explorer/pagerankTable/TableRow.js

```diff
@@ -95,3 +102,17 @@ export function PaddingRow(props: {|+backgroundColor: string|}) {
     </tr>
   );
 }
+
+export const tableRowStyle = StyleSheet.create({
+  highlightOnHover: {
+    ":hover": {
+      backgroundColor: "#EBF1F5",
+    },
+  },
+});
+
+export function setBackgroundColor(bgColor: string) {
+  return StyleSheet.create({
```

Comment:
Since aphrodite uses the `className`  prop to set styles, and inline styles uses the`style` prop, I didn't know what the effect would be of mixing them.. how/whether one would take precedence over the other, and how to pass in an effectively empty prop for one when we wanted to use the other method of setting the style. I'll re-think this.

Reply:
Also, worth mentioning for context that the green background is set dynamically, which is why the aphrodite class is created dynamically.

Reply:
Inline styles always take precedence over styles set in stylesheets;
they have maximal specificity. There’s no problem with mixing them.

> Also, worth mentioning for context that the green background is set
> dynamically, which is why the aphrodite class is created dynamically.

Right: you don’t need to do this if you use inline styles, hence the
sggestion.